docs/operations/hotplug_interfaces.md

```diff
@@ -0,0 +1,136 @@
+# Hotplug Network Interfaces
+KubeVirt now supports hotplugging network interfaces into a running Virtual
+Machine Instance (VMI). Hotplug is only supported for interfaces using the
+`virtio` model connected through
+[bridge binding](http://kubevirt.io/api-reference/main/definitions.html#_v1_interfacebridge).
+
+When a VM having hotplugged interfaces is rebooted, the hotplugged interfaces
+will be attached to the restarted VM.
+
+## Requirements
+Adding an interface to a KubeVirt Virtual Machine requires first an interface
+to be added to a running pod. This is not trivial, and has some requirements:
+
+- [multus dynamic networks controller](https://github.com/k8snetworkplumbingwg/multus-dynamic-networks-controller):
+  this daemon will listen to annotation changes, and trigger multus to configure
+  a new attachment for this pod.
+- multus running as a [thick plugin](https://github.com/k8snetworkplumbingwg/multus-cni/blob/master/docs/thick-plugin.md):
+  this multus version exposes an endpoint to create attachments for a given pod
+  on demand.
```

Comment:
nit: Please consider having Multus come first in the list.
It is the base building block of this operation.